lib/pwwka/send_message_async_sidekiq_job.rb

```diff
@@ -1,29 +1,35 @@
-require 'sidekiq'
+begin
+  require 'sidekiq'
+rescue LoadError
+end
 
 module Pwwka
   class SendMessageAsyncSidekiqJob
-    include Sidekiq::Worker
-    extend Pwwka::Logging
+    begin
+      include Sidekiq::Worker
+      extend Pwwka::Logging
 
-    sidekiq_options queue: 'pwwka_send_message_async', retry: 3
+      sidekiq_options queue: 'pwwka_send_message_async', retry: 3
 
-    def perform(payload, routing_key, options = {})
-      type = options["type"]
-      message_id = options["message_id"] || "auto_generate"
-      headers = options["headers"]
+      def perform(payload, routing_key, options = {})
+        type = options["type"]
+        message_id = options["message_id"] || "auto_generate"
+        headers = options["headers"]
 
-      logger.info("Sending message async #{routing_key}, #{payload}")
+        logger.info("Sending message async #{routing_key}, #{payload}")
 
-      message_id = message_id.to_sym if message_id == "auto_generate"
+        message_id = message_id.to_sym if message_id == "auto_generate"
 
-      Pwwka::Transmitter.send_message!(
-        payload,
-        routing_key,
-        type: type,
-        message_id: message_id,
-        headers: headers,
-        on_error: :raise,
-      )
+        Pwwka::Transmitter.send_message!(
+          payload,
+          routing_key,
+          type: type,
+          message_id: message_id,
+          headers: headers,
+          on_error: :raise,
+        )
+      end
+    rescue NameError
```

Comment:
This block needs to be wrapped in a begin/rescue as well bc both `include Sidekiq::Worker` and `sidekiq_options ...` will throw errors if line 2 isn't successful.

Reply:
Should we wrap the entire file?  I don't feel strongly, but it might be less disruptive.

Reply:
I tried that but then [this line](https://github.com/stitchfix/pwwka/blob/master/lib/pwwka/configuration.rb#L154) threw an error bc the job never ended up being defined.

Reply:
oh…right.  Yeah, that makes sense.  Carry on :)